--- pwsafe-chg_path1.orig/CMakeLists.txt	2018-02-02 22:15:36.246750194 +0100
+++ pwsafe-chg_path1/CMakeLists.txt	2018-02-02 22:20:10.420560682 +0100
@@ -392,7 +392,7 @@
 
 
 if (NOT WIN32)
-   install (DIRECTORY xml DESTINATION "share/pwsafe/")
+   install (DIRECTORY xml DESTINATION "share/passwordsafe/")
    ADD_CUSTOM_COMMAND(OUTPUT ${CMAKE_BINARY_DIR}/pwsafe.1.gz
      COMMAND ${GZIP} ARGS -9 -n -c ${PROJECT_SOURCE_DIR}/docs/pwsafe.1 > pwsafe.1.gz
 		 WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
@@ -404,7 +404,7 @@
             DESTINATION "share/locale")
 
    install (DIRECTORY ${PROJECT_BINARY_DIR}/help/
-     DESTINATION "share/doc/passwordsafe/help"
+     DESTINATION "share/passwordsafe/help"
      FILES_MATCHING PATTERN "*.zip"
      PATTERN "CMakeFiles" EXCLUDE)
 
