--- pwsafe-1.04BETA.orig/CMakeLists.txt	2018-02-08 23:13:11.783951499 +0100
+++ pwsafe-1.04BETA/CMakeLists.txt	2018-02-08 23:42:08.784588362 +0100
@@ -9,6 +9,9 @@
   cmake_policy(SET CMP0069 NEW)
 endif (POLICY CMP0069)
 
+# Allow overriding the default install location.
+set(MAN_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/share/man")
+
 # prerequisites
 include(CheckLibraryExists)
 include(CheckIncludeFile)
